addons/binding/org.openhab.binding.smartthings/src/main/java/org/openhab/binding/smartthings/discovery/SmartthingsDeviceData.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.smartthings.discovery;
+
+/**
+ * Mapping object for data returned from smartthings hub
+ *
+ * @author Bob Raker - Initial contribution
+ *
+ */
+
+public class SmartthingsDeviceData {
+    private String capability;
```

Comment:
It if is a DTO (data transfer object), please make all fields public and remove getters and the empty constructor.
Also move dto's into an own java package. Most bindings call that "model" or "dto".